pkg/cloudprovider/providers/vsphere/vsphere.go

```diff
@@ -343,11 +450,13 @@ func (vs *VSphere) CurrentNodeName(hostname string) (k8stypes.NodeName, error) {
 }
 
 // nodeNameToVMName maps a NodeName to the vmware infrastructure name
+// TODO: Rename this method to "convertToString"??
```

Comment:
Even I am confused with the current names.
If possible, please rename the function names. It will be much more simpler to read.

Reply:
Done